.pre-commit-config.yaml

```diff
@@ -40,8 +40,7 @@ repos:
     rev: v2.32.0
     hooks:
       - id: pyupgrade
-        # TODO: remove `--keep-runtime-typing` option
-        args: ["--py36-plus", "--keep-runtime-typing"]
+        args: ["--py37-plus", "--keep-runtime-typing"]
```

Comment:
I understanding of using it is that it changes `Dict` to `dict` however, I have found that this can cause issues. I will experiment with removing it but I remember it caused issues in the past